perf/benches/recycler.rs

```diff
@@ -1,22 +0,0 @@
-#![feature(test)]
-
-extern crate test;
-
-use solana_perf::{packet::PacketsRecycler, recycler::Recycler};
-
-use test::Bencher;
-
-#[bench]
-fn bench_recycler(bencher: &mut Bencher) {
```

Comment:
Heh, this was stolen from @ryoqun's PR: https://github.com/solana-labs/solana/pull/15320#issuecomment-781288262 and integrated into my PR.

Can we keep this around and also gather some before/after perf benchmarks for this? Something similar to https://github.com/solana-labs/solana/pull/15320#issuecomment-781804024